buzzmobile/launch/includes/sim_spawn_car.sh

```diff
@@ -1,5 +1,5 @@
 #! /bin/bash
 
-sleep 3
+sleep 15
```

Comment:
is this neeeddeeeeddddddddd??? such overhead, and we're not even using the launch script! why